pom.xml

```diff
@@ -675,8 +675,8 @@
           <version>1.0.1</version>
           <configuration>
             <cdapArtifacts>
-              <parent>system:cdap-data-pipeline[5.1.0-SNAPSHOT,6.0.0-SNAPSHOT)</parent>
-              <parent>system:cdap-data-streams[5.1.0-SNAPSHOT,6.0.0-SNAPSHOT)</parent>
+              <parent>system:cdap-data-pipeline[5.1.0-SNAPSHOT,10.0.0-SNAPSHOT)</parent>
```

Comment:
we can't guarantee forward compatibility like this. 10 -> 7

Reply:
Sounds good.